ddtrace/settings/config.py

```diff
@@ -204,6 +204,7 @@ def __init__(self):
             )
         self._x_datadog_tags_max_length = x_datadog_tags_max_length
         self._x_datadog_tags_enabled = x_datadog_tags_max_length > 0
+        self._propagate_service = asbool(os.getenv("DD_TRACE_PROPAGATE_SERVICE", default=True))
```

Comment:
@brettlangdon should we just set this to `False` for now since we're going to forego this feature at the moment but will likely want to add back in later?